templates/skeleton/README.md

```diff
@@ -38,3 +38,26 @@ npm run build
 ```bash
 npm run dev
 ```
+
+## Setup for using Customer Account API (`/account` section)
+
+### Setup public domain using ngrok
+
+1. Setup a [ngrok](https://ngrok.com/) account and add a permanent domain (ie. `https://<your-ngrok-domain>.app`).
+1. Install the [ngrok CLI](https://ngrok.com/download) to use in terminal
+1. Start ngrok using `npm run ngrok --domain=<your-ngrok-domain>.app`
```

Comment:
There's no npm script for ngrok defined, should this step just have them run ngrok directly?

Reply:
Or should we add an ngrok script?

Reply:
good catch! I will copy over the ngrok script in the doc